README.md

```diff
@@ -922,7 +922,10 @@ validator.isInstance(value, target); // Checks value is an instance of the targe
 | `@IsISIN()`                                     | Checks if the string is an ISIN (stock/security identifier).                                                                     |
 | `@IsISO8601()`                                  | Checks if the string is a valid ISO 8601 date.                                                                                   |
 | `@IsJSON()`                                     | Checks if the string is valid JSON.                                                                                              |
-| `@IsLowercase()`                                | Checks if the string is lowercase.                                                                                               |
+| `@IsLowercase()`                                | Checks if the string is lowercase.  
+| `@IsLatLong()`                                  | check if the string is a valid latitude-longitude coordinate in the format lat,long 
+| `@IsLatitude()`                                  | check if the string is a valid latitude coordinate
+| `@IsLongitude()`                                  | check if the string is a valid longitude coordinate
```

Comment:
1. string or number - `check if the string or number is a valid latitude/longitude coordinate`

2. Again formatting. There is more spaces than is necessary. Should be:
```md
| `@IsLatLong()`                                  | check if the string is a valid latitude-longitude coordinate in the format lat,long
| `@IsLatitude()`                                 | check if the string is a valid latitude coordinate
| `@IsLongitude()`                                | check if the string is a valid longitude coordinate
```